src/enclave/httpbuilder.h

```diff
@@ -101,7 +106,7 @@ namespace enclave
         const auto h = fmt::format(
           "{} {} HTTP/1.1\r\n"
           "{}"
-          "Content-Length: {}\r\n"
+          "content-length: {}\r\n"
```

Comment:
case

Reply:
HTTP headers are case-insensitive, so this is lower-cased to match the other places we produce our own headers.